node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/PrioritizableNode.java

```diff
@@ -128,6 +133,15 @@ public int compareTo(PrioritizableNode other) {
     /** Returns the allocation skew of the parent of this after adding this node to it */
     double skewWithThis() { return skewWith(node.resources()); }
 
+    private boolean lessThanHalfTheHost(PrioritizableNode node) {
```

Comment:
The method name is the opposite of what it returns

Reply:
No.